evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -271,15 +263,11 @@ def test_instantiate_with_parameters(example_graph):
     }
     component_graph.instantiate(parameters)
 
-    expected_order = [
```

Comment:
is the component ordering changing? Just wondering why we're changing these tests

Reply:
Yep, that was the change that caused us to pin the version a year and a half ago. For non-linear graphs, the order is now more breadth-first rather than depth-first. It's still a correct ordering, which is why the tests now simply assert that components that must be fit/transformed before others still are.